modules/interfaceClass.js

```diff
@@ -1,7 +1,10 @@
- import BookCRUD from "./bookCrudClass.js";
- 
- export default class Interface {
-  static renderBooks() {
+import BookCRUD from './bookCrudClass.js';
+
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place and try to fix the error manually as requested by the previous reviewer. You can fix this error by removing the `let Interface;` line as this variable is defined but never used.

Reply:
![image](https://user-images.githubusercontent.com/116701353/227978190-17599785-ba71-4396-a3a1-18e8ec9199bd.png)